src/components/BaseBlock.vue

```diff
@@ -8,41 +8,53 @@ defineProps<{
   iconTooltip?: string;
   iconHref?: string;
   loading?: boolean;
+  hideBottomBorder?: boolean;
+  label?: string;
+  labelTooltip?: string;
 }>();
 </script>
 
 <template>
   <div
     class="md:rounded-xl md:border border-y bg-skin-block-bg border-skin-border text-base"
   >
-    <h4
+    <div
       v-if="title"
-      class="px-4 pt-3 block rounded-t-none md:rounded-t-lg border-y border-t-0 border-skin-border"
-      style="padding-bottom: 12px"
+      class="px-4 pt-3 pb-[12px] flex rounded-t-none md:rounded-t-lg border-b border-skin-border justify-between"
+      :class="{ 'border-b-0': hideBottomBorder }"
```

Comment:
Can't this `border-b-0` class just be applied where needed, without a prop? This kind a feels wrong.

Reply:
Yeah that's pretty much everywhere at the moment. So instead of changing even more files I went with this instead.